pallets/gear/src/tests.rs

```diff
@@ -243,16 +243,7 @@ fn state_rpc_calls_trigger_reinstrumentation() {
         <Test as Config>::CodeStorage::update_code(code_and_id);
         /* ends here */
 
-        assert_ok!(Gear::read_metahash_impl(program_id, None));
         assert_ok!(Gear::read_state_impl(program_id, Default::default(), None));
-        assert_ok!(Gear::read_state_using_wasm_impl(
```

Comment:
cfg test apis (Pallet impl) could be removed as well

Reply:
it looks like it's all been deleted